actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/learner/util/Util.java

```diff
@@ -524,7 +524,7 @@ public static void initializeContext(Request actorMessage, String env) {
 
     ExecutionContext context = ExecutionContext.getCurrent();
     Map<String, Object> requestContext = null;
-    if (actorMessage.getContext().get(JsonKey.TELEMETRY_CONTEXT) != null) {
+    if ((actorMessage.getContext().get(JsonKey.TELEMETRY_CONTEXT) != null) && (actorMessage.getRequest().get(JsonKey.REQUESTED_BY) != null)) {
```

Comment:
condition is wrong, in case requested by is null, it will go into else and call the user org service, put the condition where it is actually calling user org service.